.idea/vcs.xml

```diff
@@ -18,7 +18,6 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
-    <mapping directory="$PROJECT_DIR$/docs/themes/book" vcs="Git" />
```

Comment:
We should remove this mapping, otherwise the idea will report an error when open the IDEA.